airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -99,6 +104,38 @@ def check_data_intervals(cls, values):
             )
         return values
 
+    def validate_context(self, dag: DAG) -> dict:
+        from airflow.timetables.base import DataInterval
```

Comment:
Why is this imported inside the function?

Reply:
I missed it. imported to the top